sdk/digitaltwins/digitaltwins/README.md

```diff
@@ -0,0 +1,104 @@
+## Azure AzureDigitalTwinsAPI SDK for JavaScript
```

Comment:
this readme looks like it was generated? We usually do `generate-metadata: false` in the swagger file and start from the template instead

Reply:
It was not generated and the generate-metadata is false.

---
In reply to: [470195188](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r470195188) [](ancestors = 470195188)